### PR TITLE
Initialize

### PR DESCRIPTION
**Pull request by hamidrezarezaei**

### CODE REVIEW

File: dadevarzan-wp-video.php [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+                    'hide_on_screen' => '',
+                    'active' => 1,
+                    'description' => '',
+                ));
+
+            endif;
+
+        }
+
+        public static function add_taxonomy()
```
Comment (ahmadrabie) [inline dadevarzan-wp-video.php]: delete it

File: dadevarzan-wp-video.php [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+        {
+
+        }
+
+        public static function load_text_domain()
+        {
+
```
Comment (ahmadrabie) [inline dadevarzan-wp-video.php]: remove extra space

File: dadevarzan-wp-video.php (hunk 1 above)
Comment (hamidrezarezaei) [reply dadevarzan-wp-video.php]: done this task

File: dadevarzan-wp-video.php (hunk 2 above)
Comment (hamidrezarezaei) [reply dadevarzan-wp-video.php]: done this task

**Pull request merged**